src/Microsoft.OData.Core/Value/ODataStreamReferenceValue.cs

```diff
@@ -43,7 +44,7 @@ public Uri EditLink
             {
                 return this.HasNonComputedEditLink
                     ? this.editLink
-                    : (this.computedEditLink ?? (this.metadataBuilder == null ? null : this.computedEditLink = this.metadataBuilder.GetStreamEditLink(this.edmPropertyName)));

```

Comment:
>this. [](start = 150, length = 5)

I cannot figure the reason to change this.

Reply:
Leftover from a partially reverted change. Fixed; thanks.

---
In reply to: [266668034](https://github.com/OData/odata.net/pull/1204#discussion_r266668034) [](ancestors = 266668034)